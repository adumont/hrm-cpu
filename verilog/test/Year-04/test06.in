05 41
50 13
45 71
52 6c
50 5f
49 3d
21 c6
05 4b
49 af
09 09
35 39
53 da
50 77
05 35
46 30
19 38
36 e0
46 55
28 09
50 5d
52 ec
17 6a
43 26
48 d4
43 e7
