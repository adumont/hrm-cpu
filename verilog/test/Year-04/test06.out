13
41
6c
71
3d
5f
4b
c6
09
af
da
39
35
77
38
30
55
e0
5d
09
6a
ec
d4
26
